src/index.ts

```diff
@@ -1,21 +1,24 @@
 import SpriteSheet from "./SpriteSheet";
 import Renderer from "./Renderer";
 import Tank from "./Tank";
+import KeyControls from "./KeyControls"
+
+
+
 
 async function init() {
     const spriteSheet = new SpriteSheet();
     const canvas = document.getElementById("canvas");
+    const controls = new KeyControls();
     const tank = new Tank();
 
     const renderer = new Renderer(canvas, spriteSheet);
     await renderer.loaded();
 
-    const speed = 64;
+    const speed = 200;
```

Comment:
Speed should not be global. It should depend on the tank. It will allow us to then have upgrades that change the speed as well as enemy tanks with different speeds etc.